group_vars/all.yml

```diff
@@ -279,3 +279,33 @@ __slurmdbd_config_default: # this is not needed if this PR is merged: https://gi
   DbdPort: 6819
   SlurmUser: "{{ __slurm_user_name }}"
   LogFile: "{{ __slurm_log_dir ~ '/slurmdbd.log' if __slurm_debian else omit }}"
+
+slurmdbd_config:
+  AuthType: auth/munge
+  AuthInfo: /var/run/munge/munge.socket.2
+  DbdHost: localhost
+  StorageHost: localhost
+  StorageLoc: "{{ slurm_database }}"
+  StoragePass: "{{ slurm_database_user_password }}"
+  StorageType: accounting_storage/mysql
+  StorageUser: "{{ slurm_database_user }}"
+  PidFile: /run/slurmdbd.pid
+  SlurmUser: slurm
+
```

Comment:
Does `LogFile` need to be defined here?

Reply:
No, because the value from `__slurmdbd_config_default` will be used